app/reducers/forms.js

```diff
@@ -22,6 +22,7 @@ export default formReducer.plugin({
         };
       case Event.SOCKET_UNREGISTRATION.SUCCESS:
       case Event.SOCKET_REGISTRATION.SUCCESS: {
+        if (!state) return undefined;
```

Comment:
`if (!state) return` should be the same if flow doesn't complain

Reply:
Didnt complain, but better off writing it like that. Fixed.